qutebrowser/browser/commands.py

```diff
@@ -933,9 +942,17 @@ def zoom(self, zoom=None, count=None, quiet=False):
                                           .format(zoom))
 
         level = count if count is not None else zoom
+
+        tab = self._current_widget()
+        if pdfjs.is_pdfjs_page(tab) and not no_pdfjs:
+            if level is None:
+                pdfjs.zoom(tab)
+            else:
+                pdfjs.zoom(tab, float(level) / 100)
```

Comment:
Wouldn't it make more sense to have this logic inside `pdfjs.zoom` instead, and make that take a percentage (or `None`)?

Reply:
@The-Compiler  About accepting a percentage, `AbstractZoom` accepts a float value too.